packages/components/src/VirtualizedList/HeaderIcon/HeaderIcon.component.js

```diff
@@ -13,8 +13,10 @@ import theme from './HeaderIcon.scss';
 function HeaderIcon({ label, sortBy, dataKey, sortDirection, columnData }) {
 	return (
 		<div className={classnames(theme['tc-header-icon'], 'tc-header-icon')}>
-			{columnData.iconName && <Icon name={columnData.iconName} title={label} />}
-			{sortBy === dataKey && <SortIndicator sortDirection={sortDirection} />}
+			<span title={label} aria-label={label}>
```

Comment:
Why not change the div for a span and merge the props ?

Reply:
i doesn't change the div by a span. I added a span to get a tooltip on the icon not on the whole div to get the same way that with the header with a string